pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java

```diff
@@ -945,12 +954,33 @@ public void completeHandleFuture() {
     }
 
     public void exceptionHandleFuture(Throwable t) {
+        if(isRetryableException(t)) {
+            long retryTime = backoff.next();
+            log.warn("[{}][{}] Failed to init transaction pending ack handler. It will be retried in {} Ms",
+                    persistentSubscription.getTopic().getName(), subName, retryTime, t);
+            transactionOpTimer.newTimeout((timeout) -> init(), retryTime, TimeUnit.MILLISECONDS);
+            return;
+        }
+        log.error("PendingAckHandleImpl init fail! TopicName : {}, SubName: {}", topicName, subName, t);
```

Comment:
If the error is not retriable, the Pending Ack Handler will go to a wrong value, every request that touches this Pending Ack Handler will get an error, right?

Reply:
Good suggestion. We can add a new exception.